packages/p2p/package.json

```diff
@@ -45,7 +45,7 @@
         "react-virtualized-auto-sizer": "^1.0.2",
         "react-window": "^1.8.5",
         "react-window-infinite-loader": "^1.0.5",
-        "sendbird": "^3.0.122",
+        "sendbird": "^3.0.126",
```

Comment:
Can you please make sure we're declaring the same version for both Core and P2P (or any packages that share this dependency)?

Reply:
I've removed the dependency completely as it's not required to be installed manually (sendbird-uikit specifies this in its own package.json).